src/utils.ts

```diff
@@ -86,6 +87,9 @@ export function getMediaType(searchee: Searchee): MediaType {
 			return MediaType.VIDEO;
 		case hasExt(searchee.files, [".rar"]):
 			if (MOVIE_REGEX.test(searchee.title)) return MediaType.MOVIE;
```

Comment:
This has a fall through which means that all .rar will be considered videos. It should last.